src/views/BridgeTransaction.vue

```diff
@@ -359,6 +368,41 @@ export default class BridgeTransaction extends Mixins(TranslationMixin, LoadingM
     return this.formatCodecNumber(this.soraNetworkFee)
   }
 
+  get isInsufficientBalance (): boolean {
```

Comment:
Could we reuse `hasInsufficientBalance` util function from `@/utils` here and in `Bridge `component?